### ISSUE TITLE
Halo 2 (update.xbe) - Works

### ISSUE THREAD

**Issue by PatrickvL** [Cxbx-Reloaded/game-compatibility#84]
_From @ergo720 on July 25, 2017 15:55_

I don't know if this is really useful and if you tell me it isn't I will close the issue. The update.xbe from Halo 2 works with corrupted graphics but crashes when it has finished. The game itself doesn't of course because it is an LTGC title. 
PAL, win10, 3ca1a354

[KrnlDebug.txt](https://github.com/Cxbx-Reloaded/Cxbx-Reloaded/files/1173974/KrnlDebug.txt)
![halo2](https://user-images.githubusercontent.com/30000751/28581353-8408d312-7151-11e7-9d5a-ba1d5f9295c0.PNG)

_Copied from original issue: Cxbx-Reloaded/Cxbx-Reloaded#521_

**Comment by PatrickvL** [comment]
_From @LukeUsher on July 25, 2017 15:58_

This crash issue is a known bug with HalReturnToFirmware() where it cannot resolve the symbolic link for the dashboard. This should be an easy to fix.

**Comment by ergo720** [comment]
The xbe dump of both the updater and Halo 2 reveals they are region-free and the allowed media are XBEIMAGE_MEDIA_TYPE_DVD_X2 - XBEIMAGE_MEDIA_TYPE_MEDIA_BOARD. Is this normal?
[Xbe.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/1270557/Xbe.txt)
[Xbe-Halo2.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/1270558/Xbe-Halo2.txt)

**Comment by LukeUsher** [comment]
XBEIMAGE_MEDIA_TYPE_DVD_X2 yes, that's the format of an XBOX DVD, all retail games should include this flag (but updates downloaded from Xbox Live will have the HDD flag instead)

XBEIMAGE_MEDIA_TYPE_MEDIA_BOARD is not normal however, I believe that's Chihiro!

**Comment by ergo720** [comment]
So that would mean if I had a Chihiro I could insert the game inside and play?! The case of the DVD reports "for Xbox only" though...

**Comment by LukeUsher** [comment]
Nah there's more to it than that. I own the PAL version of this game so I'll double check the xbe dump later. It could be that whatever method was used to dump this game modified the headers.

Some dashboards automatically patch XBEs on copy.

**Comment by ergo720** [comment]
Mine is PAL too. My softmodded-Xbox has UnleashX that indeed has an auto-patch xbe option but I don't it's able to patch on xbe's on the fly while they are transferred over FTP

**Comment by ergo720** [comment]
I think we should rename this as "Online Updater Application" since probably it's the same for all games and not specific to Halo 2 which should have a different issue

**Comment by PatrickvL** [comment]
This issue list is for specific software. Generic issues should be placed in the main project

**Comment by LukeUsher** [comment]
I agree with ergo720, this is a specific version of the Online Updater Application, which is not game specific at all, each unique Online Updater should be considered it's own title.

**Comment by ergo720** [comment]
@CakeLancelot (or some other member): can you rename this as "Online Updater Application"? Luke agrees and actually that's the name it has in the dump. I can't do it because I'm not a member
